cit3.0-web/src/utils/AppRoute/AppRoute.js

```diff
@@ -0,0 +1,71 @@
+import React, { useLayoutEffect } from "react";
+import Proptypes from "prop-types";
+import { useLocation, Route } from "react-router-dom";
+
+import PrivateRoute from "../PrivateRoute/PrivateRoute";
+import PublicLayout from "../../layouts/PublicLayout";
+
+const AppRoute = ({
+  component: Component,
+  layout,
+  protected: usePrivateRoute,
+  role,
+  claim,
+  title,
+  ...rest
+}) => {
+  const location = useLocation();
+  useLayoutEffect(() => {
+    if (window) {
+      window.scrollTo(0, 0);
+    }
+  }, [location.pathname]);
+
+  /* eslint react/prop-types: "off" */
```

Comment:
Is this due to props.children? If so, you should be able to add children to the prop types. There's an example in the flyout component.

Reply:
I can give that a try.